src/main/java/eu/decentsoftware/holograms/api/holograms/HologramLine.java

```diff
@@ -571,4 +572,8 @@ public boolean canShow(@NonNull Player player) {
         return super.canShow(player) && (parent == null || parent.getParent().canShow(player));
     }
 
+    // Could be considered hacky but it works?
+    public boolean shouldAwaitDisplay(Player player) {
+        return (type == HologramLineType.ICON || type == HologramLineType.HEAD) && player.getTicksLived() < Settings.DEFAULT_MINIMUM_TICKS_LIVED_ITEM_LINE;
```

Comment:
Why you check for head here now? Isn't the issue with floating items alone?
And if it is with heads too, then you have to add small head too...

Reply:
> Why you check for head here now? Isn't the issue with floating items alone? And if it is with heads too, then you have to add small head too...

Yeah, appears to be with all items..

Reply:
https://github.com/DecentSoftware-eu/DecentHolograms/pull/205/commits/2bf7b4134464f5d21793ee0ae6cb1251be3a445d